server/logger.js

```diff
@@ -1,7 +1,17 @@
 const winston = require('winston');
+require('winston-mongodb');
 
 const nodeEnv = process.env.NODE_ENV || 'development';
-
 winston.add(new winston.transports.Console(nodeEnv === 'test' && { silent: true }));
 
+const dbServer = process.env.MONGO_HOST || 'mongodb';
+const dbPort = process.env.MONGO_PORT || '27017';
+const dbUser = process.env.MONGO_USER || 'admin';
+const dbPassword = process.env.MONGO_PASSWORD || 'development';
```

Comment:
I would remove the default values for user/pass - for local dev they should always come from a .env, IMO including them in code is just asking for trouble like accidental misconfiguration